### PR TITLE
AG-592 Extended /auth/session to contain user role

### PR DESCRIPTION
**Pull request by mkl-**

### CODE REVIEW

**Comment by mmeisinger** [comment]
This is OK but not very generic for a framework purpose, but good enough.

However you use roles in 2 different ways. Using set_auth(roles=xxx) duplicates returning the roles within attributes. The set_auth variant should probably be reserved for framework roles.

File: src/ion/processes/ui/server.py [hunk 1]
```diff
@@ -214,7 +214,7 @@ def _get_user_info(self, actor_id, username=None):
         full_name = actor_user.contact.individual_names_given + " " + actor_user.contact.individual_name_family
 
         valid_until = int(get_ion_ts_millis() / 1000 + self.session_timeout)
-        set_auth(actor_id, username, full_name, valid_until=valid_until)
+        set_auth(actor_id, username, full_name, valid_until=valid_until, roles=actor_user.contact.roles)
```
Comment (mmeisinger) [inline src/ion/processes/ui/server.py]: Setting the roles here is not what is intended by the framework. These are Org roles.

Comment (mkl-) [inline src/ion/processes/ui/server.py]: It seems that roles=... adds this attribute to arguments.
From code for set_auth it follows that Ogr roles are not modified.
def set_auth(actor_id, username, full_name, valid_until, **kwargs):
    flask.session["actor_id"] = actor_id or ""
    flask.session["username"] = username or ""
    flask.session["full_name"] = full_name or ""
    flask.session["valid_until"] = valid_until or 0
    flask.session["attributes"] = kwargs.copy()
    flask.session["roles"] = {}
    flask.session.modified = True

Comment (mmeisinger) [inline src/ion/processes/ui/server.py]: You are right. This is not ideally implemented thoughm but ok. 
The actual intended org roles are not extracted from set_auth kwargs, so there is a name clash with attributes.roles. It's ok for now. Thanks!

**Pull request merged**